central/cve/image/datastore/internal/store/store.go

```diff
@@ -0,0 +1,19 @@
+package store
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/generated/storage"
+)
+
+// Store provides storage functionality for CVEs.
+//go:generate mockgen-wrapper
+type Store interface {
+	Count(ctx context.Context) (int, error)
+	Exists(ctx context.Context, id string, cve string, operatingSystem string) (bool, error)
+
+	Get(ctx context.Context, id string, cve string, operatingSystem string) (*storage.CVE, bool, error)
```

Comment:
It's the schema auto-generation. The idea was have the legacy id and the new pks so that we can transition for UI easily. I am discussing with UI team the ID stuff.